src/kvirc/ui/KviConsoleWindow.cpp

```diff
@@ -396,7 +393,11 @@ void KviConsoleWindow::saveProperties(KviConfigurationFile * cfg)
 
 void KviConsoleWindow::getBaseLogFileName(QString & buffer)
 {
-	buffer = QString("CONSOLE_%1").arg(context()->id());
```

Comment:
> still means I need more coffee... yes youre right, it shouldnt affect logging but it does for channels and queries so Im wondering if this plays nice the same way.

No, it's not like channel/query windows. Look at `KviChannelWindow::getBaseLogFileName` and `KviQueryWindow::getBaseLogFileName`, they query `windowName()` directly. I guess I could change it to do that but why?

> were talking about the case were they arent closed or no?

OK, but when would that ever be helpful? What would adding that number in the filename allow you to do? As far as I can see it's just clutter that'll get in the way of sorting.